src/data-loaders/HttpDataLoader.ts

```diff
@@ -30,19 +34,25 @@ const paramsSerializer = (params) => {
 };
 
 export default class HttpDataLoader {
-  constructor(config) {
+  private config: any;
+
+  constructor(config: any) {
     this.config = config;
   }
 
-  async load(req) {
+  async load(req: {
+    resourcePath: string;
+    params: { headers: any; query: any; body: any };
+    method: string;
+  }) {
     try {
       const { data, status, headers } = await axios.request({
         baseURL: this.config.endpoint,
-        validateStatus: false,
+        validateStatus: () => false,
```

Comment:
I'm just running into this, but what is the intention of setting this to a function that returns false? It causes any HTTP datasources to fail outright. If we want it to just allow the result of all requests we want it to return true instead of false. I believe the original intention was that it set it to false to "disable" validating status, which I guess doesn't work anymore.

Reply:
Good catch, i didn't ran into this issue, you're right it should return true
```suggestion
        validateStatus: () => true,
```